Back-End/src/main/resources/application.properties

```diff
@@ -1 +1,7 @@
 spring.application.name=Back-End
+server.port=9090
+
+amadeus.api.key=Q7tnYFeoMtIJyxaGr3n0U75T6c7kvxTr
```

Comment:
this should be here, try to send this as an env variable to the container of find a safer way to do it. sensitive info should be in the repository

Reply:
Fixed with environment variables